app/models/organization_page.rb

```diff
@@ -1,11 +1,13 @@
 class OrganizationPage < ActiveRecord::Base
   include Translatable
   include LanguageValidation
-  include Unversioned
-
-  has_paper_trail ignore: [:language]
+  include Versioning
 
   belongs_to :organization
+  has_many :organization_page_versions, dependent: :destroy
+
+  has_paper_trail ignore: [:language]
```

Comment:
Can we remove this ? I don't think anyone is using it.

Reply:
Yeah I'm removing all of them in a different PR.